src/routes/+page.svelte

```diff
@@ -1,23 +1 @@
-<script>
-  import '$lib/style.css'
-  import Banner from '$lib/Banner.svelte'
-
-  const categories = {
-    analytics: function () {
-      console.info('Dropped analytics cookies')
-    },
-    necessary: function () {
-      console.info('Dropped necessary cookies')
-    }
-  }
-  const choices = {
-    analytics: function () {
-      console.info('Dropped analytics cookies')
-    },
-    necessary: function () {
-      console.info('Dropped necessary cookies')
-    }
-  }
-</script>
-
-<Banner cookieName="test_gdpr" {categories} {choices} />
+<p>gdpr-cookie-consent-banner</p>
```

Comment:
what happened here? how do we test the banner?

Reply:
Previously I was importing from `/static` but we're no longer exporting to static.

Reply:
Resolved 6e7380ecb1aff92a6fb16a2f9ae2f515d6480a09